src/app/components/normal-component/chat/chat.component.spec.ts

```diff
@@ -252,6 +254,22 @@ describe('ChatComponent', () => {
             const chatDiv: DebugElement = testUtils.findElement('#chatDiv');
             expect(component.scrollTo).toHaveBeenCalledWith(chatDiv.nativeElement.scrollHeight);
         }));
+        it('should not loadChatContent each time user is updated twice', fakeAsync(() => {
+            // Given a chat component, hence subscribed to the current user
+            const userDAO: UserDAO = TestBed.inject(UserDAO);
+            void userDAO.set(UserMocks.CREATOR_MINIMAL_USER.id, UserMocks.CREATOR);
+            tick();
+            ConnectedUserServiceMock.setUser(UserMocks.CREATOR_AUTH_USER, true);
+            testUtils.detectChanges();
+
+            // When the user update to change its last_changed time
+            spyOn(component, 'loadChatContent').and.callThrough();
```

Comment:
ici le callThrough est inutile car on veut pas qu'elle soit appelé, donc est-ce qu'on l'inclut quand même ?

Reply:
on mettait tout par défaut je crois, et mieux vaut ça que and.callFake(() => {}) non ?

Reply:
mettre quand la fonction doit être appelée, oui, quand elle doit pas l'être c'est whatever vu qu'elle est pas appelée :D Mais allez, you can resoudre

Reply:
ah non non, je propose de fixer le fait que on est team explicite ou team réductionnisme :D ! QUOI SOMME ON ?